src/components/button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import './Button.css';
+
+function Button({name, clickable}) {
+    let disableButton = true;
+    if(clickable === "true"){
+        disableButton = false;
+    }
+    else disableButton = true;
+    return(
+        <button className="button" disabled={disableButton} >
+            {name}
+        </button>
+    )
+}
```

Comment:
// eslint-disable-next-line react/prop-types
function Button({name, clickable}) {
    let disableButton;
    disableButton = clickable !== "true";
    return(
        <button className="button" disabled={disableButton} >
            {name}
        </button>
    )
}
export default Button;

made it a little bit more compact.

Reply:
nice addition Frank, It's also inspiring. I  ended up making it this way. 

function Button({name, clickable}) {
 return(
        <button className="button" disabled={clickable !== "true"} >
            {name}
        </button>
    )
}
export default Button;

without even having to declaring a disableButton variable